tutorial/4_hextof_workflow.ipynb

```diff
@@ -56,10 +65,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# data_path can be defined and used to store the data in a specific location\n",
-    "dataset.get(\"Gd_W110\") # Put in Path to a storage of at least 10 Gbyte free space.\n",
-    "path = dataset.dir\n",
-    "print(path)"
+    "beamtime_dir = \"/asap3/flash/gpfs/pg2/2023/data/11019101\" # on Maxwell\n",
+    "if os.path.exists(beamtime_dir) and os.access(beamtime_dir, os.R_OK):\n",
+    "    path = beamtime_dir + \"/raw/hdf/offline/fl1user3\"\n",
+    "    buffer_path = beamtime_dir + \"/processed/tutorial/\"\n",
```

Comment:
if user wants to write to this folder, this won't work but since I have already populated it with buffer files, user only needs to read that data.

Reply:
This won't work then if users want to change something in the config, and thus need to recreate the buffer files. Also, does this not somewhat defy the purpose of the tutorial, if user's don't experience the full workflow (i.e. no buffer file creation)?
One could instead put just the buffer file path always into the user directly. These are fairly small for these test datasets.

Reply:
That's true. The buffer files, being of parquet format are also quite small for the couple of runs we use in the tutorial.